packages/composite-checkout/src/components/payment-request-button.js

```diff
@@ -73,7 +73,7 @@ const ApplePayButton = styled.button`
 	width: 100%;
 	position: relative;
 
-	&::after {
+	&:after {
```

Comment:
I'm curious, what's the reason for changing this to a single colon? [I think that the double-colon is more correct](https://developer.mozilla.org/en-US/docs/Web/CSS/::after#Syntax) unless there's something to do with RTL styles that I'm not aware of?

Reply:
That's my fault. I thought I remembered a linting rule about the double colon (while more correct, the single colon is supported in all modern browsers and also adds IE support, which doesn't support double), and we don't have css linting in styled components. I just verified that the rule isn't there, so I'll revert.

Reply:
> we don't have css linting in styled components

A valuable point which I hadn't considered before!